github-react-app/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+type ButtonProps = {
+    onClick?: (e: React.MouseEvent) => void;
+    children: React.ReactNode;
+    disabled: boolean;
```

Comment:
кажется, что `onClick` должен быть обязательным, а вот `disabled` нет
```suggestion
    onClick: (e: React.MouseEvent) => void;
    children: React.ReactNode;
    disabled?: boolean;
```